app/lib/methods/loadMissedMessages.ts

```diff
@@ -1,47 +1,136 @@
 import { ILastMessage } from '../../definitions';
+import { compareServerVersion } from './helpers';
+import updateMessages from './updateMessages';
 import log from './helpers/log';
 import database from '../database';
 import sdk from '../services/sdk';
-import updateMessages from './updateMessages';
+import { store } from '../store/auxStore';
+
+const count = 50;
+
+const syncMessages = async ({ roomId, next, type }: { roomId: string; next: number; type: 'UPDATED' | 'DELETED' }) => {
+	// @ts-ignore // this method dont have type
+	const { result } = await sdk.get('chat.syncMessages', { roomId, next, count, type });
+	return result;
+};
+
+const getSyncMessagesFromCursor = async (
+	roomId: string,
+	lastOpen?: number,
+	updatedNext?: number | null,
+	deletedNext?: number | null
+) => {
+	try {
+		const promises = [];
+
+		if (lastOpen && !updatedNext && !deletedNext) {
+			promises.push(syncMessages({ roomId, next: lastOpen, type: 'UPDATED' }));
+			promises.push(syncMessages({ roomId, next: lastOpen, type: 'DELETED' }));
+		}
+		if (updatedNext && typeof updatedNext !== 'undefined') {
```

Comment:
Isn't this enough?
Since we're testing the truth, both undefined and null would be falsy.

```suggestion
		if (updatedNext) {
```

Reply:
Unless updatedNext is 0 🤔